apps/snitch_core/lib/core/data/schema/store.ex

```diff
@@ -0,0 +1,36 @@
+defmodule Snitch.Data.Schema.Store do
```

Comment:
Consider, renaming the `schema` to a more related model. Also, please put a TODO tag in here for the workaround being used here to handle general settings. The approach used now for handling may not be the best approach. We can ignore it for now, but it needs to be handled. Also, create a pivotal story for the same.